lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/impl/DefaultRemoteManagerImplementation.java

```diff
@@ -89,8 +88,7 @@
         numberOfTries,
         retryTimeout,
         LocalAddressProviderFactory.getInstance(),
-        RangeTcpPortProvider.Default);
-
+        new MessagingTransportFactory());
   }
 
   /**
```

Comment:
We want to have an injected instance with Tang, instead of calling a constructor directly. @markusweimer Is this what you intended? But, I agreed that it is somewhat confusing which constructors are not needed anymore. Do we need another annotation for this?